### DISCUSSION TITLE
Ability to edit mods on existing playlist items

### DISCUSSION THREAD

**Discussion by Joehuu** [ppy/osu#13176] [Ideas]
**Describe the new feature:** Currently there is no way to edit mods from existing items. Use case is that I want to copy a playlist and just add HD on all of them.

**Proposal designs of the feature:**

Idea 1: Make the item selectable in song select, and when selected, you can change the beatmap or mods on that item. Unselect to not override that item.

Idea 2: Add a context menu to change mods, and mods overlay will show up for that specific item.

**Comment by bdach** [comment]
The big question there is what happens for any existing scores? I guess maybe this should only be allowed until any score is set? Or is this to be possible only at copy-time?

**Comment by Joehuu** [comment]
I'm talking about before making the playlist "room".
